client/src/app/pages/sbom-list/sbom-list.tsx

```diff
@@ -159,11 +159,26 @@ export const SbomList: React.FC = () => {
             <Thead>
               <Tr>
                 <TableHeaderContentWithControls {...tableControls}>
-                  <Th {...getThProps({ columnKey: "name" })} />
-                  <Th {...getThProps({ columnKey: "supplier" })} />
-                  <Th {...getThProps({ columnKey: "published" })} />
-                  <Th {...getThProps({ columnKey: "packages" })} />
-                  <Th {...getThProps({ columnKey: "vulnerabilities" })} />
+                  <Th
+                    aria-label="name"
+                    {...getThProps({ columnKey: "name" })}
+                  />
+                  <Th
+                    aria-label="supplier"
+                    {...getThProps({ columnKey: "supplier" })}
+                  />
+                  <Th
+                    aria-label="published"
+                    {...getThProps({ columnKey: "published" })}
+                  />
+                  <Th
+                    aria-label="packages"
+                    {...getThProps({ columnKey: "packages" })}
+                  />
+                  <Th
+                    aria-label="vulnerabilities"
+                    {...getThProps({ columnKey: "vulnerabilities" })}
+                  />
```

Comment:
Although nice to have, your proposed change at `client/src/app/components/TableControls/TableHeaderContentWithControls.tsx` is fully solving the original issue, don't you think it is not necessary to add these changes? Some in the file below.